samples/ssmlAddresses.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * Generates synthetic audio from a String of SSML text.
+ *
+ * Given a string of SSML text and an output file name, this function
+ * calls the Text-to-Speech API. The API returns a synthetic audio
+ * version of the text, formatted according to the SSML commands. This
+ * function saves the synthetic audio to the designated output file.
+ *
+ * ARGS
+ * ssmlText: String of tagged SSML text
+ * outfile: String name of file under which to save audio output
+ * RETURNS
+ * nothing
+ *
+ */
+async function ssmlToAudio(ssmlText, outFile) {
+  // Creates a client
+  const client = new textToSpeech.TextToSpeechClient();
+
+  // Constructs the request
+  const request = {
+    // Select the text to synthesize
+    input: {ssml: ssmlText},
+    // Select the language and SSML Voice Gender (optional)
+    voice: {languageCode: 'en-US', ssmlGender: 'MALE'},
+    // Select the type of audio encoding
+    audioConfig: {audioEncoding: 'MP3'},
+  };
+
+  // Performs the Text-to-Speech request
+  const [response] = await client.synthesizeSpeech(request);
+  // Write the binary audio content to a local file
+  const writeFile = util.promisify(fs.writeFile);
+  await writeFile(outFile, response.audioContent, 'binary');
+  console.log('Audio content written to file ' + outFile);
+}
+// [END tts_ssml_address_audio]
+
+// [START tts_ssml_address_ssml]
+/**
+ * Generates SSML text from plaintext.
+ *
+ * Given an input filename, this function converts the contents of the input text file
+ * into a String of tagged SSML text. This function formats the SSML String so that,
+ * when synthesized, the synthetic audio will pause for two seconds between each line
+ * of the text file. This function also handles special text characters which might
+ * interfere with SSML commands.
+ *
+ * ARGS
+ * inputfile: String name of plaintext file
+ * RETURNS
+ * a String of SSML text based on plaintext input
+ *
+ */
+function textToSsml(inputFile) {
+  let rawLines = '';
+  // Read input file
+  try {
+    rawLines = fs.readFileSync(inputFile, 'utf8');
+  } catch (e) {
+    console.log('Error:', e.stack);
+    return;
+  }
+
+  // Replace special characters with HTML Ampersand Character Codes
+  // These codes prevent the API from confusing text with SSML tags
+  // For example, '<' --> '&lt;' and '&' --> '&amp;'
+  let escapedLines = rawLines;
+  escapedLines = escapedLines.replace(/&/g, '&amp;');
+  escapedLines = escapedLines.replace(/"/g, '&quot;');
+  escapedLines = escapedLines.replace(/</g, '&lt;');
+  escapedLines = escapedLines.replace(/>/g, '&gt;');
+
+  // Convert plaintext to SSML
+  // Tag SSML so that there is a 2 second pause between each address
+  const expandedNewline = escapedLines.replace(/\n/g, '\n<break time="2s"/>');
+  const ssml = '<speak>' + expandedNewline + '</speak>';
+
+  // Return the concatenated String of SSML
+  return ssml;
+}
+// [END tts_ssml_address_ssml]
+
+// [START tts_ssml_address_test]
+function main(
```

Comment:
Can you make this an outer function and rearrange the region tags per the [Sample Policies](https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/nodejs.md?cl=head#canonical-samples)? Some recent examples that follow that are [here](https://github.com/googleapis/nodejs-video-intelligence/blob/master/samples/analyze-streaming-automl-classification.js) and [here](https://github.com/googleapis/nodejs-translate/blob/translate-with-glossary-sample-fix/samples/v3beta1/translate_create_glossary_beta.js)